coalib/output/ConsoleInteraction.py

```diff
@@ -607,6 +609,113 @@ def print_actions(console_printer, section, actions, failed_actions):
     return get_action_info(section, actions[choice - 1], failed_actions)
 
 
+def apply_action(action_name,
```

Comment:
Shouldnt we have smt similar for apply action action?
Anyway, should have docstring